src/ui_basic/panel.cc

```diff
@@ -1275,7 +1322,8 @@ bool Panel::ui_textinput(const std::string& text) {
 /**
  * Draw the tooltip. Return true on success
  */
-bool Panel::draw_tooltip(const std::string& text, const PanelStyle style) {
+// static
```

Comment:
Has this comment a meaning, or is it a leftover from testing?

Reply:
I always add the `// static` comment over the definition of a static class function, because it is not apparent from the function signature that this function is static